libnmstate/nm/device.py

```diff
@@ -277,6 +277,19 @@ def _delete_device_callback(src_object, result, user_data):
                 e,
             )
             mainloop.execute_next_action()
+        elif (
```

Comment:
Why did you placed a distinction between the previous failures and this one? Both results in ignoring the error, no matter if it is software or not.

Reply:
The previous failure is completely different or am I missing something?

Reply:
They are different failure for the same purpose, therefore they seem to me to fit the same condition and not a new flow.

Reply:
Done